routes/index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Set up routes for save/publish
+*/
+
+module.exports = function(server) {
+
+  // /projects API route
+  // Handles user-level operations
+  require('./projects/get-projects.js')(server);
+
+  // /project/* API routes
+  // Handles project-level operations
+  require('./project/get-contents.js')(server);
+  require('./project/create.js')(server);
+  require('./project/delete.js')(server);
+  require('./project/publish.js')(server);
```

Comment:
Do we need a `rename` here?  We probably don't want to `delete` and `create` if that means you lose data.